source/content/glossar/bibliothek/geocat-mappings/dataset-identifier.rst

```diff
@@ -1,7 +1,9 @@
 :DCAT: :ref:`dct:identifier <dcat-dataset-identifier>`
 :XPATH: ``//gmd:fileIdentifier``
 :Value: ``gco:CharacterString/text()``
-:Format: the identifier should be of the form ``<id>@<organization-slug>``
+:Usage Notes: the internal id of the dataset is taken from the XPath ``//gmd:fileIdentifier``
+              and combined with the slug of the publishing organization to form the value of
+              ``dct:identifier``, with the format ``<id>@<organization-slug>``.
```

Comment:
I would replace  ", with the format" with "as". "format" makes me always think of a distribution format